egress/django/settings_egress.py

```diff
@@ -16,10 +16,12 @@
     'default': {
         'ENGINE': 'egress.django',
         'NAME': 'django_test',
+        'USER': 'postgres',
```

Comment:
I'd rather you made this pull in from an env var, as I don't want a user set for my testing :)